### ISSUE TITLE
Protractor support for Spinner

### ISSUE THREAD

**Issue by testgitdl** [primefaces/primeng#7867]
Hello,

We are implementing the primeng elements. When creating automated tests with protractor, we are experimenting problems on the spinners. The same problems we've noticed on the examples from: https://www.primefaces.org/primeng/#/spinner

To provide more details:
- we are able to click on the up/down spinner buttons but only once
-afterwards, protractor just waits for angular indefinitely and no other action is possible 

Here is the project: https://codesandbox.io/s/primengprotractor-cmq6s

In this example after the first click the search no longer works. Code blocked after line 23. 

Can you help with this? We also noticed that after clciking the up button this element remains somehow "focused" so probably there is a miswsing mouseMove which then does not trigger mouseOut.

Thank you!

**Comment by sbley** [comment]
That's exactly what we experienced as well. We have to xit() all our e2e tests that involve a spinner.
AFAIK, the reason the spinner is not e2e testable using Protractor is the usage of `setTimeout()` within the spinner component.
The same problem occured with the growl component and it seems it got fixed by wrapping the `setTimeout()` into `runOutsideAngular()`. See https://github.com/primefaces/primeng/issues/4681

**Comment by yelhouti** [comment]
This bug has been reintroduced with Toast

**Comment by cipryd** [comment]
Any update on this?

**Comment by mertsincan** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you reopen the issue so we can include it in our roadmap? 
Please don't forget to add your feedback as a comment after reopening the issue. These will be taken into account by us and will contribute to the development of this feature. Thanks a lot for your understanding!

Best Regards,
